.semaphore/cp_dockerfile_build.yml

```diff
@@ -24,7 +24,7 @@ global_job_config:
   prologue:
     commands:
       - checkout
-      - sem-version java 8
```

Comment:
Does it make sense to use the docker container on top of the semaphoreCI agent similar to packaging?

Reply:
no @abh1sh5k that would require lots of changes to existing setup